### DISCUSSION TITLE
User Applications

### DISCUSSION THREAD

**Discussion by HumanBot000** [nextcord/nextcord#1198] [Q&A] [answered]
Is it currently possible to add specific commands for the new user apps from discord?

**Comment by DenverCoder1** [comment]
I think you are referring to User Commands

Examples:

https://github.com/nextcord/nextcord/blob/master/examples%2Fapplication_commands%2Fcontext_menus.py

Docs:

https://docs.nextcord.dev/en/stable/interactions.html#user-commands

https://docs.nextcord.dev/en/stable/api.html#nextcord.Client.user_command

**Reply by HumanBot000** [reply]
No I mean the discord thing where you can add bots to your account and use them on any server.

**Comment by EmmmaTech** [comment, marked as answer]
The implementation is currently in PR #1181, which hasn't been merged yet. In the mean time, you can install the Nextcord package directly from that branch.
